tests/AdaptiveExpressions.Tests/BadExpressionTests.cs

```diff
@@ -387,6 +387,9 @@ public class BadExpressionTests
             Test("jPath(hello,'Manufacturers[0].Products[0].Price')"), // not a valid json
             Test("jPath(hello,'Manufacturers[0]/Products[0]/Price')"), // not a valid path
             Test("jPath(jsonStr,'$..Products[?(@.Price >= 100)].Name')"), // no matched node
+            Test("merger(json(json1))"), // should have at least two arguments
+            Test("merger(json(json1), json(jarray1))"), // arguments should all be JSON objects
```

Comment:
>merger [](start = 18, length = 6)

oh, this is a bad name.  what is wrong with merge()?

Reply:
fixed